ftag/hdf5/h5split.py

```diff
@@ -69,6 +69,17 @@ def main(args=None):
             print(f"\rProcessed {total_written:,}/{total_jets:,} jets ({pct_done:.1%})", end="")
         writer.close()
 
+    if remainder != 0:
```

Comment:
Did you try to put this in the loop above and just extend the range by 1? I hoped that would work originally

Reply:
I think part of the problem is we need to define a different output size, otherwise the final file will be initialised with the same size as all others, which means we end up with a load of '0's at the end.
Also, it does run, and it does finish writing to the file (even before this change), but it basically crashes as it does so.

Reply:
yup, got you, thanks